swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -146,7 +146,7 @@ public final class Store: ObservableObject {
     TunnelManager.shared.fetchResources(callback: callback)
     let intervalInSeconds: TimeInterval = 1
     let timer = Timer(timeInterval: intervalInSeconds, repeats: true) { [weak self] _ in
-      guard let self = self else { return }
+      guard self != nil else { return }
```

Comment:
We weakly capture self here even though we never access it to avoid a race condition where the timer closure fires after `Store` has been deallocated.